calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -6539,15 +6539,20 @@ else if (type.equals("Special event"))
 				return rl.getString("legend.key15");
 			else if (type.equals("Web Assignment"))
 				return rl.getString("legend.key16");
-			else if (type.equals("Teletutoria"))
+			else if (type.equals("Formative Assessment"))
```

Comment:
Teletutoria is now not in our code, it was introduced in Sakai and not anywhere else in the code so no Teletutoria event types could have been created in the database.